test/valgrind/valgrindTestSuite.sh

```diff
@@ -886,7 +904,6 @@ then
   done
 
   echo "---------------------------------------"
-  retval=1
```

Comment:
I think this is related with 

> Made the valgrind test suite not return error if a test case has a failing func test. We know that this happens, as the timing completely changes when running under valgrind.

I'm a bit suprissed... I think this (i.e. fails in functest not involving leak/memory errors) was the way it worked, previous to this PR.

Reply:
Change undone for a test in 60730ec